h2o-core/src/main/java/hex/Model.java

```diff
@@ -2679,6 +2680,28 @@ private InteractionPair(int v1, int v2, String[] v1Enums, String[] v2Enums) {
     }
   }
 
+    /**
+     * Uploads a binary model from a given frame.
+     * Note: binary model has to be created by the same version of H2O, import of a model from a different version will fail
+     * @param destinationFrame key of the frame containing the binary representation of the model on a local filesystem, HDFS, S3...
+     * @return instance of an H2O Model
+     * @throws IOException when reading fails
+     */
+    public static <M extends Model<?, ?, ?>> M uploadBinaryModel(String destinationFrame) throws IOException {
+        InputStream is = null;
```

Comment:
is aways null and can be removed, as well as the finally block

Reply:
fixed